_redirects

```diff
@@ -18,7 +18,8 @@
 /plotly                https://docs.rapids.ai/visualization
 /plotly.html           https://docs.rapids.ai/visualization
 /slack-invite          https://join.slack.com/t/rapids-goai/shared_invite/zt-trnsul8g-Sblci8dk6dIoEeGpoFcFOQ
-/smsl                  https://studiolab.sagemaker.aws/import/github/rapidsai-community/rapids-smsl/blob/main/rapids-smsl.ipynb
+/smsl                  https://studiolab.sagemaker.aws/import/github/rapidsai-community/rapids-smsl/blob/main/rapids-smsl.ipynb?ncid=so-othe-849406-vt27#cid=av02_so-othe_en-us
+/smsl.html             https://studiolab.sagemaker.aws/import/github/rapidsai-community/rapids-smsl/blob/main/rapids-smsl.ipynb?ncid=so-othe-849406-vt27#cid=av02_so-othe_en-us
```

Comment:
```suggestion
```

We don't need a `.html` entry for this redirect.

The other entries only need to include `.html` because of some URL changes that occurred when we switched from Jekyll to Hugo for this site.